### ISSUE TITLE
BugzillaJS no longer works in Firefox Nightly 48 because nsIIoservice.newChannel() was removed

### ISSUE THREAD

**Issue by cpeterso** [gkoberger/BugzillaJS#120]
BugzillaJS no longer works in Firefox Nightly 48 after [Firefox bug 1254752](https://bugzilla.mozilla.org/show_bug.cgi?id=1254752) removed deprecated function `nsIIoservice.newChannel()`:

```
Bootstrap error: ioservice.newChannel is not a function
readURI@resource://gre/modules/addons/XPIProvider.jsm -> jar:file:///Users/chris/Library/Application%20Support/Firefox/Profiles/3qg4hndg.Test/extensions/jid0-NgMDcEu2B88AbzZ6ulHodW9sJzA@jetpack.xpi!/bootstrap.js:51:17
startup@resource://gre/modules/addons/XPIProvider.jsm -> jar:file:///Users/chris/Library/Application%20Support/Firefox/Profiles/3qg4hndg.Test/extensions/jid0-NgMDcEu2B88AbzZ6ulHodW9sJzA@jetpack.xpi!/bootstrap.js:86:30
this.XPIProvider.callBootstrapMethod@resource://gre/modules/addons/XPIProvider.jsm:4682:9
this.XPIProvider.startup@resource://gre/modules/addons/XPIProvider.jsm:2684:13
callProvider@resource://gre/modules/AddonManager.jsm:227:12
_startProvider@resource://gre/modules/AddonManager.jsm:755:5
AddonManagerInternal.startup@resource://gre/modules/AddonManager.jsm:938:9
this.AddonManagerPrivate.startup@resource://gre/modules/AddonManager.jsm:2773:5
amManager.prototype.observe@resource://gre/components/addonManager.js:57:7
```

**Comment by mnoorenberghe** [comment]
Looks like the SDK was updated in https://bugzilla.mozilla.org/show_bug.cgi?id=1241561 so we probably just need to repack against a version that includes that fix. (It doesn't seem like we've released a new version of the SDK for a while so I guess that means trunk).

I just created a version locally (even though I don't use this extension anymore due to most of the feature being included in BMO) but Github doesn't allow xpi attachments.

**Comment by ncalexan** [comment]
I can't live without my "new" links after the Depends and Blocks fields. @mnoorenberghe would you mind uploading to people.mozilla.org?

**Comment by mnoorenberghe** [comment]
That's one of the features I miss the most but it's actually there in the Mozilla theme but with different UI:
![screen shot 2016-03-14 at 2 28 28 pm](https://cloud.githubusercontent.com/assets/601139/13760575/11496ba4-e9f1-11e5-86e3-9e4cea0f2ac9.png)


Unsigned XPI: https://people.mozilla.org/~mnoorenberghe/tmp/bugzillajs.xpi
Flip xpinstall.signatures.required to false on Nightly or Aurora to install

**Comment by mnoorenberghe** [comment]
@rik I'd be willing to help with maintenance of this extension as I'd hate to have the 1200 users abandoned. Email me MattN@MoCo if you're interested.

**Comment by gkoberger** [comment]
@mnoorenberghe I added you as a collaborator :)

**Comment by gkoberger** [comment]
(Whoops, didn't mean to close)

**Comment by mnoorenberghe** [comment]
> @mnoorenberghe I added you as a collaborator :)

Thanks @gkoberger . There wasn't actually a code change necessary for this fix, I just built using a newer SDK. Do you want me to make an XPI with a version bump to submit to AMO? My AMO email address is firstname @ lastname .com if you want to add me there.

**Comment by rik** [comment]
And I've added you as an owner on AMO. Thanks for your help!

(Side note: I'm not interacting with Bugzilla on a day-to-day basis anymore so I'm not putting the efforts to maintain this add-on)

**Comment by mnoorenberghe** [comment]
Fixed in v3.5.0: https://addons.mozilla.org/en-US/firefox/addon/bugzillajs/versions/3.5.0 with gkoberger/omnium#7
